numpy/lib/tests/test_regression.py

```diff
@@ -167,8 +167,8 @@ def test_who_with_0dim_array(self):
         try:
             try:
                 np.who({'foo': np.array(1)})
-            except Exception:
-                raise AssertionError("ticket #1243")
```

Comment:
There's very little point making these changes in the tests. Probably best to undo the test changes, simply because it makes `git blame` slightly less useful, and this error probably never will happen again.

Reply:
i undo this change, and send new commits.

Reply:
You ended up redoing this change, because after removing it, you merged with the change that added it back.

I think your mistake was not using `git push -f`.

Reply:
yes, i fix my commits again.